README.md

```diff
@@ -1,5 +1,13 @@
 # learn-layer5
 
+# To check for smi conformance of a deployed service mesh
+We use kuttl to check for SMI conformance. All the tests are writtten in smi-test directory of this repository.
+Execute the following command to run the smi-conformace tests:-
+
+```shell
+kubectl kuttl test  --skip-cluster-delete=true --start-kind=false ./smi-test
```

Comment:
Can we instead use the kuttl `harness` and `testutils` packages? - https://kuttl.dev/docs/api-integration.html

Reply:
Yeah, the same tests can be invoked using golang apis...